QModManager/API/Game.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace QModManager.API
+{
+    using System;
+
+    /// <summary>
+    /// Identifies the Subnautica games.
+    /// </summary>
+    [Flags]
+    public enum Game
```

Comment:
Could we use a less ambiguous name for this enum?

Reply:
Main issue is that `Game` is a class in the Unity assemblies.  

My only thought would be to use something like `QModGame`.